src/annotations/constants/mocks.ts

```diff
@@ -25,6 +25,29 @@ export interface AnnotationStream {
 }
 
 export const MOCK_ANNOTATION_STREAMS: AnnotationStream[] = [
+  {
+    id: 'default',
+    name: 'Default',
+    description: 'Default Annotation stream',
+    meta: {
+      createdAt: '2d ago',
+      updatedAt: '6h ago',
+    },
+    display: {
+      color: '#8e1fc3',
+      summaryCol: 'summary',
+      messageCol: 'message',
+      timeStartCol: '_timeStart',
+      timeStopCol: '_timeStop',
+    },
+    query: {
+      tags: {
+        _field: 'deployments',
+      },
+      measurement: 'annotations',
+      bucketName: 'defBuck',
+    },
+  },
```

Comment:
might as well remove the other annotation mocks since they're not used and they're irrelevant and confusing